client/locales/de/translation.json

```diff
@@ -23,6 +23,8 @@
     "changesSaved": "Die Änderungen wurden gespeichert.",
     "failedToSaveChanges": "Die Änderung konnten nicht gespeichert werden.",
     "loginFailed": "Ihr Benutzername oder Passwort ist unbekannt.",
+    "loginTimeout": "Server braucht zu lange um zu antworten.",
+    "genericServerError": "Es ist ein Problem auf dem Server aufgetreten.",
```

Comment:
Maybe add smth like "Server braucht zu lange um zu antworten. Wir arbeiten daran. Bitte versuch es später nochmals." and "Es ist ein Problem auf dem Server aufgetreten. Wir arbeiten daran. Bitte versuch es später nochmals."

So the user knows that we are working on it and that they should come back later again.